src/mono/mono/metadata/assembly.c

```diff
@@ -2038,7 +2043,7 @@ mono_assembly_request_open (const char *filename, const MonoAssemblyOpenRequest
 			return NULL;
 		}
 
-		if (load_req.asmctx != MONO_ASMCTX_REFONLY) {
```

Comment:
Is there any reason to leave this block?

Reply:
*shrug* wanted to leave the indentation unchanged